frontend/src/utils/storybook.tsx

```diff
@@ -32,6 +33,23 @@ export const LoggedInDecorator: DecoratorFn = (storyFn) => {
   return storyFn()
 }
 
+export const ViewedFeatureTourDecorator: DecoratorFn = (
+  storyFn,
+  { parameters },
+) => {
+  const userId = parameters.userId
+  useEffect(() => {
+    window.localStorage.setItem(
+      FEATURE_TOUR_KEY_PREFIX + userId,
+      JSON.stringify(true),
+    )
+
+    return () => window.localStorage.removeItem(LOGGED_IN_KEY)
```

Comment:
seems like we are removing the wrong key here.

Reply:
Updated as per your comment